[2025-10-14T18:28:36.003+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: events_analytics_pipeline.dbt_docs_generate scheduled__2025-10-14T17:00:00+00:00 [queued]>
[2025-10-14T18:28:36.005+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: events_analytics_pipeline.dbt_docs_generate scheduled__2025-10-14T17:00:00+00:00 [queued]>
[2025-10-14T18:28:36.005+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-10-14T18:28:36.009+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): dbt_docs_generate> on 2025-10-14 17:00:00+00:00
[2025-10-14T18:28:36.012+0000] {standard_task_runner.py:60} INFO - Started process 3053 to run task
[2025-10-14T18:28:36.014+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'events_analytics_pipeline', 'dbt_docs_generate', 'scheduled__2025-10-14T17:00:00+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/events_pipeline_dag.py', '--cfg-path', '/tmp/tmp7qod5spt']
[2025-10-14T18:28:36.015+0000] {standard_task_runner.py:88} INFO - Job 167: Subtask dbt_docs_generate
[2025-10-14T18:28:36.032+0000] {task_command.py:423} INFO - Running <TaskInstance: events_analytics_pipeline.dbt_docs_generate scheduled__2025-10-14T17:00:00+00:00 [running]> on host 7fd364930f15
[2025-10-14T18:28:36.058+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='events_analytics_pipeline' AIRFLOW_CTX_TASK_ID='dbt_docs_generate' AIRFLOW_CTX_EXECUTION_DATE='2025-10-14T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-14T17:00:00+00:00'
[2025-10-14T18:28:36.059+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-14T18:28:36.059+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/event_gen && /home/***/.local/bin/dbt docs generate --profiles-dir /opt/***/.dbt --project-dir /opt/***/event_gen']
[2025-10-14T18:28:36.066+0000] {subprocess.py:86} INFO - Output:
[2025-10-14T18:28:36.743+0000] {subprocess.py:93} INFO - [0m18:28:36  Running with dbt=1.10.13
[2025-10-14T18:28:36.829+0000] {subprocess.py:93} INFO - [0m18:28:36  Registered adapter: clickhouse=1.9.4
[2025-10-14T18:28:36.957+0000] {subprocess.py:93} INFO - [0m18:28:36  Found 4 models, 30 data tests, 1 source, 599 macros
[2025-10-14T18:28:36.959+0000] {subprocess.py:93} INFO - [0m18:28:36
[2025-10-14T18:28:36.959+0000] {subprocess.py:93} INFO - [0m18:28:36  Concurrency: 4 threads (target='dev')
[2025-10-14T18:28:36.959+0000] {subprocess.py:93} INFO - [0m18:28:36
[2025-10-14T18:28:37.261+0000] {subprocess.py:93} INFO - [0m18:28:37  Building catalog
[2025-10-14T18:28:37.312+0000] {subprocess.py:93} INFO - [0m18:28:37  Catalog written to /opt/***/event_gen/target/catalog.json
[2025-10-14T18:28:38.339+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-14T18:28:38.355+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=events_analytics_pipeline, task_id=dbt_docs_generate, execution_date=20251014T170000, start_date=20251014T182836, end_date=20251014T182838
[2025-10-14T18:28:38.385+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-14T18:28:38.395+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
